e2e/testsuite/testsuite.go

```diff
@@ -484,6 +491,39 @@ func (s *E2ETestSuite) ExecuteGovProposal(ctx context.Context, chain *cosmos.Cos
 	s.Require().Equal(govtypes.StatusPassed, proposal.Status)
 }
 
+
+// QueryModuleAccountAddress returns the sdk.AccAddress of a given module name.
+func (s *E2ETestSuite) QueryModuleAccountAddress(ctx context.Context, moduleName string, chain *cosmos.CosmosChain) (sdk.AccAddress, error) {
+	authClient := s.GetChainGRCPClients(chain).AuthQueryClient
+
+	moduleAccountsResponse, err := authClient.ModuleAccounts(ctx, &authtypes.QueryModuleAccountsRequest{})
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Should we use this [grpc](https://github.com/cosmos/cosmos-sdk/blob/main/x/auth/keeper/grpc_query.go#L141) instead?